src/components/editMeasure/populationCriteria/QDMReporting/QDMReporting.test.tsx

```diff
@@ -0,0 +1,223 @@
+import * as React from "react";
+import { render, fireEvent, waitFor, screen } from "@testing-library/react";
+import { act } from "react-dom/test-utils";
+import useMeasureServiceApi, {
+  MeasureServiceApi,
+} from "../../../../api/useMeasureServiceApi";
+import { Measure } from "@madie/madie-models";
+import QDMReporting from "./QDMReporting";
+
+const mockHistoryPush = jest.fn();
+
+jest.mock("react-router-dom", () => ({
```

Comment:
not sure if routeHandlerStore requires this to be mocked, but I don't see a direct usage of react router

Reply:
removed